site/tutorials/keptn-notification-service.md

```diff
@@ -0,0 +1,277 @@
+summary: Create a Keptn Notification Service
+id: keptn-notification-service
+categories: tutorial
+tags: tutorial
+status: Draft
+authors: Adam Gardner
+Feedback Link: https://keptn.sh
+
+# Create a Custom Keptn Notification Service
+
+## Welcome
+Duration: 00:01:00
+
+This tutorial is a hands-on guide which will demonstrate how to write a custom Keptn "notification" service.
+
+If you'd rather follow the official documentation, it's [here](https://keptn.sh/docs/0.6.0/usecases/custom-service/). If you prefer a Go-based example, my colleague Christian Kreuzberger gave a [video walkthrough on Youtube](https://www.youtube.com/watch?v=42762GuB6C0).
+
+## Prerequisites
+Duration: 00:01:00
+
+- A Keptn installation (full installation or quality gates only)
+- A Docker Hub account
+- Docker installed (and authenticated to DockerHub) on your local machine
+
+## Notification Service Workflow
+Duration: 00:02:00
+
+Keptn "notification" services are standard services, they're known as "notification" services because they only listen and react to events.
+They do not influence Keptn's workflow in any way.
+
+Keptn services can be written in any language you choose.
+
+A "notification" service workflow is something like this:
+
+1. A distributor listens for a particular event
+2. The distributor distributes the event to the subscribing service on a provided port
+3. The custom service ingests the event
+4. The custom service performs some logic (that you code)
+
+## Create the Keptn Distributor
+Duration: 00:02:00
+
+Every Keptn service requires a corresponding distributor. A distributor is a component which subscribes to Keptn events and forwards them your service. Think of a distributor as the "glue" between the Keptn core engine and your service.
+
+Save the following content on to the Keptn machine as a file called `demo-service-distributor.yaml`:
```

Comment:
I am afraid the term "keptn machine" is a bit confusing. probably just name it local machine?